brainsets/datasets/OdohertySabesNonhuman2017.py

```
@@ -5,6 +5,32 @@
+    **Brain Regions:** M1, S1
+
+    **Dataset Statistics**
+
+    - **Subjects:** 2
+    - **Total Sessions:** 47
+    - **Total Units:** 16,566
+    - **Events:** ~105.2M spikes, ~12.4M behavioral timestamps
+
+    **Links**
+
+    - Paper: `O'Doherty and Sabes (2018) – Journal of Neural Engineering <https://pubmed.ncbi.nlm.nih.gov/29192609/>`_
```

Ruff (0.15.5)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)